File: examples/advanced-crud/src/main/java/org/example/advancedcrud/app/repository/DirectorRepository.java
package org.example.advancedcrud.app.repository;

import org.example.advancedcrud.app.entity.Director;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

public interface DirectorRepository extends JpaRepository<Director, Long>, JpaSpecificationExecutor<Director> {
}

File: examples/basic-crud-kotlin/src/main/kotlin/org/example/basiccrudkotlin/app/repository/DirectorRepository.kt
package org.example.basiccrudkotlin.app.repository;

import org.example.basiccrudkotlin.app.entity.Director
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import org.springframework.data.rest.core.annotation.RepositoryRestResource


@RepositoryRestResource(path = "directors")
interface DirectorRepository : JpaRepository<Director, Long>, JpaSpecificationExecutor<Director> {
}